docs/src/content/docs/manual-setup.mdx

```diff
@@ -0,0 +1,128 @@
+---
+title: Manual Setup
+description: Learn how to configure Starlight manually to add it to an existing Astro project.
+---
+
+import { Tabs, TabItem } from '@astrojs/starlight/components';
+
+The quickest way to create a new Starlight site is using `create astro` as shown in the [Getting Started guide](/getting-started/#creating-a-new-project).
+If you want to add Starlight to an existing Astro project, this guide will explain how.
+
+## Set up Starlight
+
+To follow this guide, you’ll need an existing Astro project.
+
+### Add the Starlight integration
+
+Starlight is an [Astro integration](https://docs.astro.build/en/guides/integrations-guide/). Add it to your site by running the `astro add` command in your project’s root directory:
+
+<Tabs>
+    <TabItem label="npm">
+        ```sh
+        npx astro add starlight
+        ```
+
+    </TabItem>
+    <TabItem label="pnpm">
+        ```sh
+        pnpm astro add starlight
+        ```
+    </TabItem>
+    <TabItem label="Yarn">
+        ```sh
+        yarn astro add starlight
+        ```
+    </TabItem>
+
+</Tabs>
+
+This will install required dependencies and add Starlight to the `integrations` array in your Astro config file.
```

Comment:
Missing article
```suggestion
This will install the required dependencies and add Starlight to the `integrations` array in your Astro config file.
```